peps/pep-0772.rst

```diff
@@ -0,0 +1,463 @@
+-----------
+
+The Python Steering Council would be expected to delegate decision making to
+the Packaging Council for PEPs related to the Python packaging.  The bodies
+would work together on issues that intersect the packaging domain and language
+stewardship (including the CPython implementation, standard library, and
+distribution).
+
+The PSF Board would be encouraged to formally deprecate the Packaging Workgroup
+and the Packaging Council would wholly take on the responsibilities of the
+PSF's Packaging Workgroup.
+
+The PyPA would be expected to work with the Packaging Council to establish a
+decision making process that governs the technical projects under the PyPA
+umbrella.
+
+Processes
+=========
+
+Election of the council
+-----------------------
+
+A council election consists of two phases:
+
+* Phase 1: A member of the Packaging community (defined later in this document)
+  can self-nominate themselves for the council elections.
+* Phase 2: Each core team member can assign zero to five stars to each
+  candidate. Voting is performed anonymously. The outcome of the vote is
+  determined using the `STAR voting system`_, modified to use the `Multi-winner
+  Bloc STAR`_ approach. If a tie occurs, it may be resolved by mutual agreement
+  among the candidates, or else the winner will be chosen at random.
+
+Each phase should last two weeks.
+
+The election process is managed by a returns officer nominated by the outgoing
+Packaging Council. For the initial election, the returns officer will be
+nominated by the PSF Executive Director.
+
+The PSF shall maintain records of the elections as well as run the annual
+election for the council.
+
+Term
+----
+
+There shall be two cohorts of council members: Cohort A composed of two (2)
+members and Cohort B composed of three (3) members.
+
+Each council member shall be elected for a two-year term, unless they are
+replacing a council member that resigned, was removed, or otherwise becomes
+vacant, in which case such replacement members shall be elected to a term
+sufficient to complete a two-year term as measured from the term of the
+original cohort.
+
+Each council member's term will be two (2) years, from when their elections are
+finalised until the next elections for their cohort are finalised.
+
+Only for elections involving the entire council (like the initial council
+election), the two candidates receiving the highest number of votes shall be
+designated Cohort A with a two (2) year term, and the three candidates
+receiving the highest number of votes after shall be designated Cohort B with a
+one (1) year term.
+
+There are no term limits for individual council members.
+
+Vacancies
+---------
+
+Council members may resign their position at any time.
+
+Whenever there is a vacancy during the regular council term, the council may
+vote to appoint a replacement to serve out the rest of the term.
+
+If a council member drops out of touch and cannot be contacted for a month or
```

Comment:
Should the phrasing account for scheduled out of touch time periods? If someone is intentionally away for over a month and has notified the rest of the council, would this affect them?

```suggestion
If a council member unexpectedly drops out of touch and cannot be contacted for a month or
```

I understand that this is based on PEP 13, but that's not a reason to not improve/clarify if possible - and then update PEP 13 as well.

Reply:
> If someone is intentionally away for over a month and has notified the rest of the council, would this affect them?

I would think not, but hopefully the member and the council can work together to discuss whether an extended, but in-touch absence should necessitate a resignation, in the best interests of the community.

> I understand that this is based on PEP 13, but that's not a reason to not improve/clarify if possible - and then update PEP 13 as well.

There's definitely a [process for that](https://peps.python.org/pep-0013/#changing-this-document), and there have been [3 amendments to date](https://peps.python.org/pep-0013/#history-of-amendments).

Reply:
I'm inclined to say that this isn't an editorial concern but a concern with the specifics of the proposal -- and that we should discuss any such specifics in the corresponding d.p.o. thread.

I've taken note of this FWIW, and will make sure to flag this in that discussion.